tasks/utils/artifact.js

```diff
@@ -4,51 +4,54 @@ const fs = require("node:fs/promises")
 const currentVersion = "0"
 
 // export interface RequestData {
-//  // location of source code (only Inline is currently supported)
-//   codeLocation: Location.Inline,
-//  // location of secrets (Inline or Remote)
-//   secretsLocation: Location.Inline,
-//  // code language (only JavaScript is currently supported)
-//   codeLanguage: CodeLanguage.JavaScript,
-//  // string containing the source code to be executed
-//   source: fs.readFileSync("./calculation-example.js").toString(),
-//  // secrets can be accessed within the source code with `secrets.varName` (ie: secrets.apiKey). The secrets object can only contain string values.
-//   secrets: { apiKey: process.env.COINMARKETCAP_API_KEY ?? "" },
-//  // Redundant URLs which point to encrypted off-chain secrets
-//   secretsURLs: [],
-//  // args (string only array) can be accessed within the source code with `args[index]` (ie: args[0]).
-//   args: ["1", "bitcoin", "btc-bitcoin", "btc", "1000000", "450"],
-//  // expected type of the returned value
-//   expectedReturnType: ReturnType.uint256,
-//  // Default offchain secrets object used by the `functions-build-offchain-secrets` command
-//   globalOffchainSecrets: {},
-//  // Per-node offchain secrets objects used by the `functions-build-offchain-secrets` command
-//   perNodeOffchainSecrets: [],
+//   type: 'consumer' | 'automatedConsumer'
+//   id: string;
```

Comment:
Notice that requests can either be:
type: 'consumer', which use request id as their id
or 
type: 'automatedConsumer', which use the AutomatedConsumer contract address as their id

Reply:
What is the best way to make this clear to the user?  Could we have `requestId` and `automationContractAddress` instead? Maybe we could even separate Automation requests into a separate file from regular requests?

I think it may be confusing that `id` has different meanings in different contexts.

Reply:
I agree. Broke them apart into two different folders.